pkg/manifests/template/utils.go

```diff
@@ -4,7 +4,7 @@ import (
 	console "github.com/pluralsh/console-client-go"
 )
 
-func configMap(svc *console.ServiceDeploymentExtended) map[string]string {
+func configMap(svc *console.GetServiceDeploymentForAgent_ServiceDeployment) map[string]string {
```

Comment:
Could we have a more straightforward type for this?

Reply:
Unfortunately, that's autogenerated by the `gqlgenc` package we use in `console-client-go`. The new, v2 client now generates struct names differently. It's `<queryName>_<returnType>`.